modules/hmmer/hmmsearch/meta.yml

```diff
@@ -0,0 +1,69 @@
+name: hmmer_hmmsearch
+description: search profile(s) against a sequence database
+keywords:
+  - hidden Markov model
```

Comment:
```suggestion
  - hidden markov model```

Reply:
Normally I would capitalize Markov since it's a given name but I guess keywords should be all lowercase?